src/components/Exchange/FeesTooltip.tsx

```diff
@@ -0,0 +1,122 @@
+import { i18n, MessageDescriptor } from "@lingui/core";
+import { defineMessage, Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import Tooltip from "components/Tooltip/Tooltip";
+import { getConstant } from "config/chains";
+import { BigNumber, BigNumberish } from "ethers";
+import { useChainId } from "lib/chains";
+import { USD_DECIMALS } from "lib/legacy";
+import { formatAmount, formatAmountFree } from "lib/numbers";
+
+type Fee = { label: string; value: string };
+type ExecutionFee = { fee?: BigNumberish; feeUSD?: BigNumberish };
+type FeeType = "open" | "close" | "swap" | "borrow" | "deposit" | "execution";
+
+const FEE_LABELS: Record<FeeType, MessageDescriptor> = {
```

Comment:
not critial, but, i think, what if just wrap these messages to function like:
```
function getFeeLabel(type: FeeType) {
  const labels = {
     close: ...
     open: ...
   }
    
  return labels[type]
}
```
Then it will work when the language is changed? Just to avoid a bit complicated syntax with defineMessage and 18n._

Reply:
Updated!